fineract-provider/src/main/java/org/apache/fineract/infrastructure/bulkimport/importhandler/loan/LoanImportHandler.java

```diff
@@ -464,6 +464,7 @@ private Integer importLoanRepayment(final List<LoanTransactionData> loanRepaymen
         JsonObject loanRepaymentJsonob = gsonBuilder.create().toJsonTree(loanRepayments.get(rowIndex)).getAsJsonObject();
         loanRepaymentJsonob.remove("manuallyReversed");
         loanRepaymentJsonob.remove("numberOfRepayments");
+        loanRepaymentJsonob.remove("reversed");
```

Comment:
what is this?

Reply:
During this part of the loan import we create a json object made from LoanTransactionData objects, however there are certain fields that the loan import validates for, that cannot exist in the json object during import. Reversed is one of these fields and since I added it to the LoanTransactionData at one point the tests related to loan imports started failing. I found this part, where we do the same with the "manuallyReversed" and "numberOfRepayments" fields. However since I removed the reversed field from LoanTransactionData we don't need this. I'll remove it.